### DISCUSSION TITLE
Performance test based on clsx

### DISCUSSION THREAD

**Discussion by tordans** [dcastil/tailwind-merge#252] [Ideas]
It would be awesome to have the performance test that clsx uses run for `twMerge` and `twJoin` and have a page in the repo that shows the results.

- Benchmark results by clsx https://github.com/lukeed/clsx/tree/master/bench
- Benchmark linked by clsx and classnames https://github.com/JedWatson/classnames/tree/main/benchmarks

I found existing references to performance, but in the end they do not really tell me how the libraries compare.
- https://github.com/dcastil/tailwind-merge/discussions/243 says "349 calls to twMerge with a total of 5.1 ms execution time." but how does this compare the the clsx/classnames benchmar?
- https://github.com/dcastil/tailwind-merge/discussions/164 explains very well why it is fast, but not how fast
- https://github.com/dcastil/tailwind-merge/blob/v1.13.1/docs/features.md#performance does the same (and probably should reference twJoin as well)

**Comment by dcastil** [comment]
Hey @tordans! 👋 

Yeah that makes sense, I want to do that rather sooner than later. But will take some time since I'm going to be unavailable for at least two weeks and need to find time for it afterwards.

**Reply by dcastil** [reply]
I added a new issue to track this: https://github.com/dcastil/tailwind-merge/issues/253
